framework/test/jest/unit/specs/modules/chain/bft/bft.spec.js

```diff
@@ -583,6 +583,129 @@ describe('bft', () => {
 			});
 		});
 
+		describe('#isBFTProtocolCompliant', () => {
+			let bft;
+			let blocks;
+
+			beforeEach(async () => {
+				bft = new BFT(bftParams);
```

Comment:
An `// Arrange` comment at the beginning of the hook would be nice.
```suggestion
				// Arrange
				bft = new BFT(bftParams);
```